vmdb/db/migrate/20100308005125_create_miq_cim_instances.rb

```diff
@@ -14,7 +14,7 @@ def self.up
       t.column :agent_id,         :integer
       t.column :stat_id,          :integer
       t.column :stat_top_id,        :integer
-      t.timestamps
+      t.timestamps :null => true
```

Comment:
If we want everything to go the new way, then this PR should get merged, and there should be a data migration to fix the database to be accurate in the "new" way.